README.md

```diff
@@ -268,7 +270,7 @@ cd offchain
 cargo build
 ```
 
-Now you can run the onchain tests alongside the state servers by running the following commands:
+Now you can run the on-chain tests alongside the state servers by running the following commands:
```

Comment:
This tests only refer to the rollups testing, right?  When we say on-chain code testing, it should also include test for directories as `rollups-arbitration `and `rollups-cli.` I'd rename the `on-chain` references and write `rollups` instead as is the main directory we're testing.

Reply:
I see your point. I guess we have two things being called the same, the entire solution (repo) is the `rollups` repo that is composed by` off-chain` and `on-chain`, the `on-chain` part is composed by `rollups`, `rollups-arbitration` and `rollups-cli`. I could understand leaving `onchain tests` as it's going to be as ambiguous change it to `rollups` tests, but definitely the name `on-chain tests `only include `rollups contracts` and not `arbitration` and `cli`.

Reply:
Yes, the naming is unfortunate. But don't worry, we have to refactor this whole README anyway, because we aren't even testing the on-chain alongside the state servers anymore... So we can take this opportunity to also make this naming less ambiguous. :-)